marlowe-playground-server/src/Server.hs

```diff
@@ -90,23 +90,34 @@ app handlers =
 
 newtype AppConfig = AppConfig {authConfig :: Auth.Config}
 
-initializeContext :: IO AppConfig
-initializeContext = do
+initializeServerContext :: Maybe FilePath -> IO AppConfig
+initializeServerContext secrets = do
+  putStrLn "Initializing Context"
```

Comment:
i suppose we don't have a `MonadLogger` at this point right?

Reply:
exactly